src/components/posts/Facebook/socialMedia/AdDisplay.js

```diff
@@ -20,5 +23,33 @@ export default function AdDisplay(props) {
         </p>
       </div>
     );
-  return <div id="shown-ad">Will display the {selectedAd} ad</div>;
+  return (
+    <div id="shown-ad">
```

Comment:
Should the outline of our shown-ad section look the same as the generate profile section?
<img width="711" alt="Screen Shot 2021-07-21 at 9 49 32 PM" src="https://user-images.githubusercontent.com/65370631/126591308-34e1e7ce-629b-4c89-bd75-44e9ede902a1.png">